src/boot/bootctl.c

```diff
@@ -1523,6 +1568,18 @@ static int parse_argv(int argc, char *argv[]) {
                         assert_not_reached();
                 }
 
+        if ((arg_root || arg_image) && !STRPTR_IN_SET(argv[optind], "status", "list", "install", "update",
+                        "remove", "is-installed", "random-seed"))
+                return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
+                                       "Options --root= and --image= are not supported with verb %s.",
```

Comment:
I wonder what happens when there is no verb. Does it crash or does it print unhelpful messages with `(n/a)/(null)`? More generally, why are verbs analyzed here? I thought that's what `dispatch_verb` is for

Reply:
depends on libc used. glibc will output (null), other libcs might crash.

so yes, thia call should cover for a NULL verb better.